ws-nextjs-app/pages/_app.page.tsx

```diff
@@ -67,6 +67,8 @@ export default function App({ Component, pageProps }: Props) {
     isUK,
   } = pageProps;
 
+  const { metadata: { atiAnalytics } = {} } = pageData;
```

Comment:
I don't think the live page response has a metadata object, the atiAnalytics just lives in the pageData

Reply:
Ahh sorry didn't see your FABL PR